src/main/java/com/wyft/services/RideService.java

```diff
@@ -12,31 +12,17 @@ public RideService (RideRepository rideRepository){
 		this.rideRepository = rideRepository;
 	}
 
-	public void createRide(Ride rideToCreate){
-		rideRepository.addRide(rideToCreate);
-	}
-
-	public int acceptRide(Driver driver, Integer acceptedRideID){
-	    //check the logic for whether or not a ride can be accepted, then invoke a repository method to accept it
+	public void createRide(Ride rideToCreate){ rideRepository.addRide(rideToCreate); }
 
-		return -999;
-	}
+	//create a method called acceptRide
+	//it must check the logic for whether or not a ride can be accepted, then invoke a repository method to accept it
```

Comment:
question do we want to firmly affix the logic for checking the ride to the service?  In one interview I was in candidate wanted to run some checks in the controller and only invoke service if the ride was acceptable.  Might it make sense to allow for this level of flexibility?  The only problem is how do you convey that would be ok?  Or is it not?

Reply:
I think we should leave the comments as is, because usually "business logic" should stay in the controller. If a candidate has good reason to do something else, we can be open to change.